node/bundler.ts

```diff
@@ -167,5 +175,22 @@ const getBasePath = (sourceDirectories: string[], inputBasePath?: string) => {
   return commonPathPrefix(sourceDirectories)
 }
 
+export const addGeneratorFieldIfMissing = (
+  declarations: Declaration[],
+  functions: EdgeFunction[],
+  internalFunctionsPath?: string,
+) =>
+  declarations.map((declaration) => {
+    const fullFuncPath = functions?.find((func) => func.name === declaration.function)?.path
+
+    // If function path is in the internalFunctionsPath, we assume it is autogenerated.
+    const isInternal = Boolean(
+      internalFunctionsPath && fullFuncPath && isPathInside(fullFuncPath, internalFunctionsPath),
+    )
+
+    const generatorFallback = isInternal ? 'internalFunc' : undefined
```

Comment:
Curious why you're going with "internal" here if we're saying that these are "generated" functions?

Reply:
Also, if the property is "generator" and the value is "internalFUNC", does it mean that the function generates itself? 😅 

Would it make sense to use something like "netlify"?

Reply:
I've thought about this. So I wanted to keep it similar to what I did in zip-it-and-ship-it. And essentially when prepopulating, we don't know if it is netlify. we also don't know if it was a plugin or integration. We only know that it was placed in an internal folder. It's a bit icky though, but that is the thought behind it I guess 🤔 I _could_ change it to `plugin/integration` or `netlify`